core/src/main/scala/kafka/admin/DeleteRecordsCommand.scala

```diff
@@ -61,26 +64,31 @@ object DeleteRecordsCommand {
     if (duplicatePartitions.nonEmpty)
       throw new AdminCommandFailedException("Offset json file contains duplicate topic partitions: %s".format(duplicatePartitions.mkString(",")))
 
+    val recordsToDelete = offsetSeq.map(offset => (offset._1, RecordsToDelete.beforeOffset(offset._2))).toMap.asJava
```

Comment:
Use `{ case (a, b) =>` (choose appropriate variable names) to avoid the ugly `_1` and `_2`